src/dask_awkward/utils.py

```diff
@@ -149,3 +149,23 @@ def second(seq: Iterable[T]) -> T:
     the_iter = iter(seq)
     next(the_iter)
     return next(the_iter)
+
+
+def field_access_like(entry: Any) -> bool:
+    if isinstance(entry, str):
+        return True
+    if isinstance(entry, (list, tuple)) and all(isinstance(x, str) for x in entry):
+        return True
+    return False
+
+
+def field_access_to_front(seq: Sequence[Any]) -> tuple[Any, ...]:
```

Comment:
@martindurant this is the function we talked a bit about yesterday.

Reply:
If you invert it:
```python
def field_access_like(entry: object) -> bool:
    if isinstance(entry, str):
        return False
    if isinstance(entry, (list, tuple)) and all(isinstance(x, str) for x in entry):
        return False
    return True
```

Then it's simply:
```
sorted(seq, key=field_access_like)
```
because it will put "field-likes" first but otherwise not change ordering.

Reply:
(well, this would be a list version)